src/uct/tcp/tcp_ep.c

```diff
@@ -100,7 +96,9 @@ static UCS_CLASS_CLEANUP_FUNC(uct_tcp_ep_t)
     ucs_list_del(&self->list);
     UCS_ASYNC_UNBLOCK(iface->super.worker->async);
 
-    ucs_free(self->buf);
+    if (self->length != 0)
```

Comment:
coding style: plz always use `{}` for `if` `else` statements

Reply:
done